packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.android.js.snap

```diff
@@ -35,19 +30,22 @@ exports[`Android API levels Android API<21 should render without ripple 1`] = `
     onStartShouldSetResponder={[Function]}
     style={
       Array [
-        Array [
-          Object {
-            "alignItems": "center",
-            "borderRadius": 40,
-            "flexDirection": "row",
-            "justifyContent": "center",
-            "paddingHorizontal": 12,
-            "paddingVertical": 8,
-          },
-        ],
+        Object {
+          "alignItems": "center",
+          "flexDirection": "row",
+          "justifyContent": "center",
+          "paddingHorizontal": 12,
+          "paddingVertical": 8,
+        },
+        Object {
+          "borderRadius": 40,
+        },
         Object {
           "backgroundColor": "rgb(0, 215, 117)",
         },
+        Object {
+          "flexDirection": "row-reverse",
```

Comment:
Is this expected?

Reply:
Yeah - it's because I changed the component internally slightly. Before it was like this:

```
<outer>
  <text />
  <icon />
</outer>
```

Now it's like this:

```
<outer>
  <icon />
  <text />
</outer>
```

So when `iconAlignment=trailing` I'm setting `row-reverse` on the outer part to make the icon appear after the text